README.adoc

```diff
@@ -14,3 +14,145 @@ We would love to hear from you.
 
 If you want to collaborate in the project, we would also love to get your Pull Requests. Before you
 start working on one, please take a look at our link:CONTRIBUTING.adoc[collaboration manual].
+
```

Comment:
Three paragraphs up, the paragraph starting with "Currently, this repository contains..." needs to be updated too.